shanoir-uploader/ShanoirUploader.jnlp

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?> 
+<jnlp spec="1.0+" codebase="https://github.com/fli-iam/shanoir-ng/releases/latest/download/" href="ShanoirUploader.jnlp">
+	<information>
+		<title>Shanoir Uploader</title>
```

Comment:
Hi @LaurentPV, thank you for this great PR!
Could you please write ShanoirUploader in one word?
Thank you and with kind regards, Michael